internal/pkg/term/color/color.go

```diff
@@ -82,11 +82,16 @@ func HighlightResource(s string) string {
 	return HiBlue.Sprint(s)
 }
 
-// HighlightCode wraps the string with the ` character, colors it to denote it's a code block, and returns it.
+// HighlightCode wraps the string s with the ` character, colors it to denote it's code, and returns it.
 func HighlightCode(s string) string {
 	return HiCyan.Sprintf("`%s`", s)
 }
 
+// HiglightCodeBlock wraps the string s with ``` characters, colors it to denote it's a multi-line code block, and returns it.
+func HiglightCodeBlock(s string) string {
```

Comment:
Ooh-la-la!

Reply:
We can now update our help menus and recommended actions with this fn, it'll print code much nicer